src/styles/EmailForm.css

```diff
@@ -0,0 +1,36 @@
+.emailform-notification-title{
```

Comment:
can this go inside the one below that has the same class name?

Reply:
That  is  a different  class. I should be header instead of title. I fixed it